### DISCUSSION TITLE
Missed orders when calling GET /orders/v0/orders

### DISCUSSION THREAD

**Discussion by d3stozaur** [amzn/selling-partner-api-models#3946] [General]
When calling for order list on GET /orders/v0/orders with parameters as this
https://sellingpartnerapi-eu.amazon.com/orders/v0/orders?MarketplaceIds=A1PA6795UKMFR9,A13V1IB3VIYZZH,AMEN7PMS3EDWL,APJ6JRA9NG5V4,A1805IZSGTT6HS,A1C3SOZRARQ6R3,A2NODRKZP88ZB9&LastUpdatedAfter=2024-06-05T22:25:26Z&OrderStatuses=Unshipped,PartiallyShipped,Shipped,InvoiceUnconfirmed,PendingAvailability,Pending&MaxResultsPerPage=100

We get a list of 100 orders but no order with id 402-1508113-9189104 but when we add AmazonOrderIds parameter 
https://sellingpartnerapi-eu.amazon.com/orders/v0/orders?MarketplaceIds=A1PA6795UKMFR9,A13V1IB3VIYZZH,AMEN7PMS3EDWL,APJ6JRA9NG5V4,A1805IZSGTT6HS,A1C3SOZRARQ6R3,A2NODRKZP88ZB9&LastUpdatedAfter=2024-06-05T22:25:26Z&OrderStatuses=Unshipped,PartiallyShipped,Shipped,InvoiceUnconfirmed,PendingAvailability,Pending&AmazonOrderIds=402-1508113-9189104&MaxResultsPerPage=100

we get that order. We tried changaing LastUpdateAfter value, replacing it with CreatedAfter parameter, removing OrderStatuses parameter but nothing works we are unable to get the order 402-1508113-9189104 with /orders/v0/orders without passing it in the AmazonOrderIds parameter. Why this order isnt returned in the first request?

**Comment by shreeharsh-a** [comment]
Please reach out to [developer support](https://developer.amazonservices.com/support) so that we can better help you with issues related to SP API.

`Note: SP API or docs related issues / troubleshooting support is managed by a different team. We can help if the issue is related to the content published on this repo.`
